docs/troubleshooting.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Troubleshoot known issues"
+description: "Describes the known issues and troubleshooting you can do to resolve these issues for a .NET Multi-platform App UI (.NET MAUI) app."
+ms.date: 05/25/2022
+---
+
+# Troubleshooting known issues guide
+
+This article describes some of the known issues with .NET MAUI, and how you can solve or work around them. The [.NET MAUI code repository](https://github.com/dotnet/maui/wiki/Known-Issues) also details some known issues.
+
+[!INCLUDE [docs under construction](~/includes/preview-note.md)]
+
+## Templates are missing
+
+If you've installed Visual Studio 2022 and the **.NET Multi-platform App UI development** workload, but the .NET MAUI templates are missing, you most likely have a conflict with a .NET 7 preview version. To see if .NET 7 is being resolved as your current version of .NET, perform the following steps:
+
+01. Open a terminal.
+01. Run the `dotnet --version` command.
+
+    If the result starts with `7.0`, you're running in the context of .NET 7. Use one of the fixes below.
+
+> [!TIP]
+> You can see all versions of .NET installed with the `dotnet --info` command.
+
+### Fix (1)
+
+Uninstall the .NET 7 preview.
+
+### Fix (2)
```

Comment:
Worth stressing somewhere in this section that it's a project level fix (i.e. has to be done for every project)?